messages/main/en.json

```diff
@@ -9,6 +9,7 @@
   "menu-file": "File",
   "menu-change-language": "Change language...",
   "menu-change-language-title": "Enter the language (e.g., 'en' or 'es')",
+  "menu-config-complete": "New config has been loaded",
```

Comment:
Based on https://github.com/digidem/mapeo-desktop/issues/618, message should be `Successfully imported config:`